### ISSUE TITLE
[Feature]: Vampire Armor Config 

### ISSUE THREAD

**Issue by SonRaon** [TeamLapen/Vampirism#1366]
### What?

We all know that vampire players have natural armor that doesn't stack with any worn armor the vampire players may have.

My idea is to add a config option in the Vampirism configs folder that allows the players to choose if they'd like to activate vampire armor stacking or not.

### Mod experience

The change I'm proposing doesn't effect the balancing indirectly, unless someone **actively** chooses to alter that specific config, due to the fact that the vampire armor stacking config should be on _false_ by default.

It wouldn't remove the vampire movement speed and attack speed penalties since the main idea behind the config/feature I'm proposing is that it would allow vampires to reap the benefits of both vampire armor and the protection values of their desirable armor, but it wouldn't remove any other penalties.

It'd also help vampire players to fair better in Modpacks that have mods that allow for higher armor attribute values like Attribute Fix since it'd help said players to have a better alternative, until they get the highly pricey - in terms of crafting resources - armor set they're after in said Modpack and would smooth out their play style against the harder hitting mobs of other mods.

**Comment by SonRaon** [comment]
@maxanier apologies in advance for the tag

**Comment by lunofe** [comment]
@SonRaon If you apologise in advance, why would you do it 😐
Edit: You've also apologised after doing the same in https://github.com/mystchonky/Tome-of-Blood/issues/8, drop that behaviour.

**Comment by SonRaon** [comment]
@lunofe I apologized in advance in case my message had found the mod author when they were doing something important.

Being polite with everyone isn't a bad thing to do.

**Comment by kaboom24** [comment]
I agree with this since without it you don't get armor toughness which is a HUGE nerf. It also just makes sense,  especially with how easily hunters can destroy vampires
